aetest/instance_vm.go

```diff
@@ -39,6 +39,10 @@ func NewInstance(opts *Options) (Instance, error) {
 			i.startupTimeout = opts.StartupTimeout
 		}
 	}
+	if len(os.Getenv("APPENGINE_DEV_APPSERVER_BINARY")) > 0 {
+		os.Setenv("APPENGINE_DEV_APPSERVER", os.Getenv("APPENGINE_DEV_APPSERVER_BINARY"))
```

Comment:
You should move this to `startChild` and avoid rewriting the env var (unless this `Setenv` is really necessary, maybe I'm missing something?)